### ISSUE TITLE
Plugging in PS5 controller messes up inputs.

### ISSUE THREAD

**Issue by Zarfoid** [Ryochan7/DS4Windows#2256]
When playing wireless all of the buttons work like they should, but as soon as I plug in the controller, they bug out and the x button works as the circle. This started happening after the recent update, not sure if I can fix it. Already tried reinstalling.

**Comment by mika-n** [comment]
try older versions one version at the time to see if the problem popped up in some specific version.
However, the current devel version (not yet released) does have some mapping bug fixes, so maybe those fix the issue. Wait for the new release.
https://github.com/Ryochan7/DS4Windows/releases

**Comment by IKanraI** [comment]
I've had this issue before but it was when I went into the controller profile and edited it to be a ps4 controller

Inside profile edit > other > emulated controller 

Change this setting to Xbox 360 if it's set to ps4 and it is what fixed it for me. The game I noticed it doing it on was Resident Evil 5 and it doesn't do it for every game for me. e.g. persona 4 golden accepts the dualshock 4 emulated controller mapping for the game and doesn't rearrange the buttons -- might have something to do with how re5 is tied to windows play center? Not sure
